entries/normalizer.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<entry name="normalizer" type="method" return="String">
+	<title>normalizer</title>
+	<desc>Change/transform the value of the element before validating.</desc>
+	<longdesc>
+		The user can change the value of an element before validating the element in question. The new value will be then used by the associated methods instead of the real one.
```

Comment:
I will copy pasta your comment :D (without the comment about the server side, as it's obvious)